### PR TITLE
Pull my finger

### PR DESCRIPTION
**Pull request by Saulis**
[Review on Reviewable](https://reviewable.io/reviews/saulis/components-1/1)

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="http://app.review.ninja/Saulis/components-1/pull/1" target="_blank">ReviewNinja

**Comment by Saulis** [comment]
[CLA assistant check](https://cla-assistant.io/Saulis/components-1?pullRequest=1) <br/>All committers have accepted the CLA.

**Comment by Saulis** [comment]
<sup>**[.travis.yml, line 3 [r2]](https://reviewable.io:443/reviews/saulis/components-1/1#-Jp1oPgmbcyM5nXyxI59)** ([raw file](https://github.com/saulis/components-1/blob/50e5c7f7204242984b2d2b8012a8b5410fcdb872/.travis.yml#L3)):</sup>
Java 8 go go go

---

<sup>**[.travis.yml, line 6 [r2]](https://reviewable.io:443/reviews/saulis/components-1/1#-Jp1oW2U5-OPk583nqT1)** ([raw file](https://github.com/saulis/components-1/blob/50e5c7f7204242984b2d2b8012a8b5410fcdb872/.travis.yml#L6)):</sup>
css is nice also

---

Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/saulis/components-1/1)

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+language: java
+install: npm install
+jdk: oraclejdk8
```
Comment (Saulis) [inline .travis.yml]: nice

**Comment by Saulis** [comment]
<sup>**[.travis.yml, line 3 [r2]](https://reviewable.io:443/reviews/saulis/components-1/1#-Jp1oEyTyFDPywoWFNfw-r2-3)** ([raw file](https://github.com/saulis/components-1/blob/50e5c7f7204242984b2d2b8012a8b5410fcdb872/.travis.yml#L3)):</sup>
Done.

---

<sup>**[.travis.yml, line 3 [r2]](https://reviewable.io:443/reviews/saulis/components-1/1#-Jp1oPgmbcyM5nXyxI59)** ([raw file](https://github.com/saulis/components-1/blob/50e5c7f7204242984b2d2b8012a8b5410fcdb872/.travis.yml#L3)):</sup>
Done.

---

<sup>**[.travis.yml, line 6 [r2]](https://reviewable.io:443/reviews/saulis/components-1/1#-Jp1oW2U5-OPk583nqT1)** ([raw file](https://github.com/saulis/components-1/blob/50e5c7f7204242984b2d2b8012a8b5410fcdb872/.travis.yml#L6)):</sup>
Done.

---

Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/saulis/components-1/1)

**Comment by Saulis** [comment]
Reviewed files:
- .travis.yml @ r2

---

Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/saulis/components-1/1)

**Pull request closed without merging**